source/transactions-convenient-api/transactions-convenient-api.md

```diff
@@ -154,16 +167,32 @@ This method should perform the following sequence of actions:
 This method can be expressed by the following pseudo-code:
 
 ```typescript
+var BACKOFF_INITIAL = 1  // 1ms initial backoff
```

Comment:
Let's update this to use 5ms as described in the tech doc. And the growth rate as well.